### PR TITLE
bugfixes: create_query is_private field name & add optional arguments to run_sql()

### PR DESCRIPTION
**Pull request by agaperste**
- fixing create_query's is_private param 
- adding performance param for run_sql

### CODE REVIEW

**Comment by agaperste** [comment]
Hi @bh2smith @RichardKeo, when you get a chance can you take a look? Tried to address 1 error and 1 request here. I tested manually and both functions work. Thanks in advance!

File: .pylintrc [hunk 1]
```diff
@@ -1,2 +1,4 @@
 [MASTER]
-disable=fixme,logging-fstring-interpolation
\ No newline at end of file
+disable=fixme,logging-fstring-interpolation
+[DESIGN]
+max-args=10
```
Comment (bh2smith) [inline .pylintrc:4]: There are probably a few ignore statements that can now be removed.

**Review by bh2smith — APPROVED** [review]
Looks good, but what is up with the title of this PR? It's just a bunch of meaningless codes (I'm sure they mean something, but this public repo has no connection to your foreign task manager).

I strongly suggest renaming that before merge. Think about what you would expect people to see in the release notes (e.g. https://github.com/duneanalytics/dune-client/releases/tag/v1.3.0)


Would be nice to remove some pylint ignores in other places too.

File: dune_client/api/query.py [hunk 2]
```diff
@@ -33,7 +33,7 @@ def create_query(
         payload = {
             "name": name,
             "query_sql": query_sql,
-            "private": is_private,
+            "is_private": is_private,
```
Comment (msf) [inline dune_client/api/query.py:36]: this should have a test, but this fix is correct

**Review by msf — APPROVED** [review]

**Comment by msf** [comment]
I've fixed the commit message. Yes, we shouldn't pollute the commit history of open source repo with the task-manager issue numbers.

**Pull request merged**